modules/bitgo/src/v2/coins/sol.ts

```diff
@@ -123,7 +123,7 @@ export class Sol extends BaseCoin {
   }
 
   async verifyTransaction(params: SolVerifyTransactionOptions): Promise<any> {
```

Comment:
please update unit tests that cover token transfer + creation transactions.
see `modules/bitgo/test/v2/unit/coins/sol.ts`

Reply:
Updated the test, will cover the `verifyTransaction` for send token once the sendMany fixed, so the test can support multiple token as well (kinda line expensive to cover one by one since need to specify new `TxParam` and `TxPrebuild`)